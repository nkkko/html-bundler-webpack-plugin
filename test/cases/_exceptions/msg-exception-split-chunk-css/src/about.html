<!DOCTYPE html>
<html>
<head>
  <title>About</title>
  <link href="./css/style.scss" rel="stylesheet">
  <script src="./js/main.js" defer="defer"></script>
</head>
<body>
  <h1>About</h1>
</body>
</html>