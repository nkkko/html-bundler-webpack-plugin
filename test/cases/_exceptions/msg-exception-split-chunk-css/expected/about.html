<!DOCTYPE html>
<html>
<head>
  <title>About</title>
  <link href="css/style.2eb0c66b.css" rel="stylesheet">
  <script src="js/main.7693a3d5.js" defer="defer"></script>
</head>
<body>
  <h1>About</h1>
</body>
</html>